core/tests/puppeteer-acceptance-tests/user-utilities/logged-in-users-utils.ts

```diff
@@ -674,10 +674,11 @@ export class LoggedInUser extends BaseUser {
       throw new Error('The Watch A Video button does not exist!');
     }
     await Promise.all([
-      this.page.waitForNavigation(),
+      this.page.waitForNavigation({waitUntil: ['load', 'networkidle2']}),
```

Comment:
If we already have this waitUntil, do we still need the new function waitForText in order to fix the flake?